string nav_goal
string grasp
string place
---
bool finish